api/src/migrations/1729164610693-initial-functions-and-triggers.ts

```diff
@@ -55,8 +18,8 @@ export class InitialFunctionsAndTriggers1729164610693
         p_slug varchar;
         p_name varchar;
         p_description varchar;
-        widget_ids int[] := '{}';
-        BEGIN
+        widget_indicators text[] := '{}';
```

Comment:
Overall, the whole function can be removed (should be) as we will be using the crafted sections json with attached widgets, and the repository to handle CRUDs over this. 

Once this is addressed:
https://github.com/Vizzuality/4-growth/pull/137/files#r1811990729

We can remove it